docs/getting-started/typescript-options-api.md

```diff
@@ -9,246 +9,144 @@ position: 30
 ---
 
 
-# Get Started with Kendo UI for Vue Native Components with TypeScript and Options API
+# Get Started with Kendo UI for Vue
 
-In this article you’ll learn how to use Kendo UI for Vue Native components in TypeScript context. Below we are building a small app that includes a Grid, a DropDownList, a Window and a design theme.
+This tutorial will help you develop a simple app that includes a native Vue Data Grid component. To achieve this, you will build a project using [Vite](https://vitejs.dev/) and the [Vue Composition API](https://github.com/telerik/kendo-vue/tree/master/getting-started-javascript-options-api) paired with TypeScript.
+
+>Curious about JavaScript or the Composition API? This tutorial comes in several additional variants:
+>* [Kendo UI for Vue with TypeScript and the Composition API](slug:getting_started_typescript_composition_api)
+>* [Kendo UI for Vue with JavaScript and the Options API](slug:getting_started_javascript_options_api)
 
-> Historically, all Kendo UI for Vue Native components support both **Vue 2** and **Vue 3**, however, Vue 2 is currently in its end-of-support phase till Nov 2024. After our last major release for 2024, Vue 2 will no longer be supported in the new versions of the Kendo UI for Vue components. Please check our [Vue 2 End of Support article](https://www.telerik.com/kendo-vue-ui/components/vue2-deprecation/) for more details.
 
-The current example uses the recommended by Vue [Vite build tool](https://vitejs.dev/) + [Vue Options API](https://vuejs.org/guide/introduction.html#options-api) + Typescript. If you need Getting started information that uses `Vue Composition API` + Typescript, you can check [this link](slug:getting_started_typescript_composition_api).
+> Historically, all Kendo UI for Vue Native components support both **Vue 2** and **Vue 3**, however, Vue 2 is currently in its end-of-support phase till Nov 2024. After our last major release for 2024, Vue 2 will no longer be supported in the new versions of the Kendo UI for Vue components. Please check our [Vue 2 End of Support article](https://www.telerik.com/kendo-vue-ui/components/vue2-deprecation/) for more details.
 
 > Prefer video tutorials? How about a free Telerik UI onboarding course? Check out the [Kendo UI for Vue with TypeScript](https://learn.telerik.com/learn/course/internal/view/elearning/45/kendo-ui-for-vue-with-typescript) training in [Telerik Virtual Classroom](https://learn.telerik.com/learn).
```

Comment:
Again, there are too many callouts in a row, diluting the focus. The following approach might make sense for spreading them throughout the page (this is applicable to the other pages as well).

* The video tutorials callout could be placed right under the H1. The idea being that you might be looking for a video tutorial and skip reading this one altogether.
* The Vue 2 note should be probably around the step about the framework selection (when creating the project) or in the section about adding the data grid component.

I am open to discussion around the details, but we need to move these pieces around to improve the reading experience.

Reply:
I moved the video tutorial callout under the H1